kas-installer.sh

```diff
@@ -57,6 +57,17 @@ read_kas_installer_env_file() {
   fi
 }
 
+check_observability_token() {
+  curl --silent \
+    --head \
+    --header "Accept: application/vnd.github+json" \
+    --header "Authorization: Bearer ${OBSERVABILITY_CONFIG_ACCESS_TOKEN}" \
+    ${OBSERVABILITY_CONFIG_REPO} \
+    --fail \
+  | grep -i 'github-authentication-token-expiration' \
```

Comment:
I think this might not work for non-expiring tokens.

Reply:
Yep, grep exited 1 on no matches. TIL. I've pushed it up with the grep removed and pushing the curl output to `/dev/null`.